connector/connector_ldap.go

```diff
@@ -217,11 +238,74 @@ func (c *LDAPConnector) TrustedEmailProvider() bool {
 	return c.trustedEmailProvider
 }
 
+type LDAPPool struct {
+	m          sync.Mutex
+	conns      map[*ldap.Conn]*ldap.Conn
```

Comment:
The empty struct takes [zero memory](https://play.golang.org/p/BR2urbScRB). But I wouldn't worry about memory constraints since you'll have at most a 100 connections or so.

I suggested this because `map[*ldap.Conn]*ldap.Conn` looks odd. This just seems more explicit. Wouldn't say it's the norm but I have seen maps combined with the empty struct for things like this.